docs/epayment.v1.yml

```
@@ -244,8 +243,12 @@ components:
             See: https://developer.vippsmobilepay.com/docs/vipps-developers/common-topics/vipps-landing-page/
           minLength: 10
           maxLength: 15
-          example: "4791234567"
+          example: '4791234567'
           pattern: '^\d{10,15}$'
+        personalQr:
+          type: string
+          description: 'The personal QR from the app, used in merchant scan scenarios. Pass on all information from the QR, do not format it in any way.'
```

Attempt at making it clear. We should probably have a `oneOf` in the API spec for https://developer.vippsmobilepay.com/api/epayment/#tag/CreatePayments/operation/createPayment with both `phoneNumber` and something like `personalQrCode`, so we can validate both variants properly.

```suggestion
          description: 'The user's personal QR code in the app. This QR code can be scanned by merchants, and used in the `POST:/epayment/v1/payments` request to initiate a payment. **Important:** The content of the QR code can change at any time, without warning, so it is important that the _entire content of the QR code_ is specified as the `phoneNumber` when initiating payments. Do not attempt to extract the phone number from the QR code contents, as the content may change and no longer contain a phone number.'
```

---

should this be in the description instead?
We shouldnt mention that the phonenumber is part of the QR at all.. I can try to rephrase it

---

I'm DM-ing @khenrichsen, and he will clean this up. 🧹